### DISCUSSION TITLE
Maq

### DISCUSSION THREAD

**Discussion by mouhand999** [community/community#123038] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

تم اختراق جميع أجهزتي و هذا خطاء سبب لي المشاكل الكثيرة و تم تغيير جميع أرقامي السرية من قبلهم و لازلت أواجه بعض المشاكل

**Comment by LiteBrite82** [comment, marked as answer]
Hi @mouhand999 , thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
